content/docs/csidriver/release/powerflex.md

```diff
@@ -2,20 +2,15 @@
 title: PowerFlex
 description: Release notes for PowerFlex CSI driver
 ---
+## Release Notes - CSI PowerFlex v2.13.1
 
-## Release Notes - CSI PowerFlex v2.13.0
-
-
-
-
-
-
-
-
-
+### New Features/Changes
 
+### Fixed Issues
+- [#1782 - [BUG]: Pods Stuck in Terminating State After PowerFlex CSI Node Pod Restart When Deployments Share Same Node](https://github.com/dell/csm/issues/1782)
 
 
+## Release Notes - CSI PowerFlex v2.13.0
```

Comment:
There should be a known issue added to 2.13, no?

Reply:
I've updated the Known Issue on PowerFlex v2.12.0 for csm-v1.12.0 here: https://github.com/dell/csm-docs/pull/1478/commits/5c2242c0cf4f92781496c699f070d98fcd71c5c9. I'll check with the team and get back, but I think it needs to be added here as well.